GDJS/Runtime/runtimescene.ts

```diff
@@ -78,6 +78,29 @@ namespace gdjs {
       this.onGameResolutionResized();
     }
 
+    /**
+     * Activate or deactivate the debug visualization for collisions and points.
+     * @memberof gdjs.RuntimeScene
+     */
+    renderDebugDraw(
```

Comment:
render means "calling this function will render things on screen". 
But this should not do it! There is already a function that is called `renderAndStep` in runtime scene.
Here, you said your function is "Activate or deactivate the debug visualization for collisions and points.". So it should:
1) be called `enableDebugDraw`
2) it should not call a function called `renderDebugDraw` :) It should just set on or off some booleans. Then, in `renderAndStep`, the runtime scene should call `renderDebugDraw` if it's enabled.

Calling clearDebugDraw is fine, because it's a one time operation.